docs/README.rst

```diff
@@ -0,0 +1,138 @@
+.. _readme:
+
+letsencrypt-formula
+===================
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/letsencrypt-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/letsencrypt-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+Creates certificates and manages renewal using the letsencrypt service.
+
+.. contents:: **Table of Contents**
+
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
+
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+If you need (non-default) configuration, please pay attention to the ``pillar.example`` file and/or `Special notes`_ section.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see `How to contribute <https://github.com/saltstack-formulas/.github/blob/master/CONTRIBUTING.rst>`_ for more details.
+
+Special notes
+-------------
+
+None
+
+Available states
+----------------
+
+.. contents::
+   :local:
+
+``letsencrypt``
+^^^^^^^^^^^^^^^
+
+This is a shortcut for letsencrypt.install letsencrypt.config and letsencrypt.domains.
+
+If `use_package` is `True` (the default), the formula will try to install the *certbot* package from your Distro's repo.
+Keep in mind that most distros don't have a package available by default: Ie, current Debian (Stretch) requires a backports repo installed.
+Centos 7 requires EPEL, etc. This formula **DOES NOT** manage these repositories. Use the `apt-formula <https://github.com/saltstack-formulas/apt-formula>`_
+or the `epel-formula <https://github.com/saltstack-formulas/epel-formula`_ to manage them.
```

Comment:
This section has carried across from the original `README`.  This PR currently adds supports for the EPEL repo; perhaps that needs to be adjusted.  The two options to select from:

1. Follow this `README` and use the `epel-formula` from the CI only, removing the commit for the EPEL repo.
1. Leave the commit for the EPEL repo and update the `README` instead.